### PR TITLE
resources: augeas: Make augeas support optional

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: docs/documentation.md [hunk 1]
```diff
@@ -558,14 +558,28 @@ no binary available continue working in a temporary directory to avoid failure.
...
 
+#### Disable augeas support
+
+If you wish to compile mgmt without augeas support, you can use the following command:
+
+```
+GOTAGS=noaugeas make build
+```
+
+### Combining compile-time flags
```
Comment (purpleidea) [inline docs/documentation.md]: Patch looks perfect, except I think this section needs a fourth ```#```. If not, then the general comment expressed here should be in the `###` "Compilation options" section.

Comment (roidelapluie) [inline docs/documentation.md]: fixed

**Comment by purpleidea** [comment]
ACK

**Comment by purpleidea** [comment]
Maintainers guilt + insomnia == Merged! w00t!

**Pull request merged**